src/Controller/Api/AuthController.php

```
@@ -0,0 +1,416 @@
+    {
+        $expireSeconds = $rememberMe ? 2592000 : 86400; // 30 days or 24 hours
+        $isSecure = 'prod' === ($_ENV['APP_ENV'] ?? 'dev');
+
+        return Cookie::create(self::REFRESH_TOKEN_COOKIE)
+            ->withValue($token)
+            ->withExpires(time() + $expireSeconds)
+            ->withPath('/')
+            ->withHttpOnly(true)
+            ->withSecure($isSecure)
+            ->withSameSite('strict');
+    }
```

PHPMD (2.15.0)

createRefreshTokenCookie accesses the super-global variable $_ENV. (undefined)

(Superglobals)